### PR TITLE
boards: remove header guard under scores

### PR DESCRIPTION
**Pull request by OlegHahm**
This replaces #2858

### CODE REVIEW

File: boards/mbed_lpc1768/system.c [hunk 1]
```diff
@@ -382,34 +382,34 @@
...
 
 /* F_cco0 = (2 * M * F_in) / N  */
-#define __M               (((PLL0CFG_Val      ) & 0x7FFF) + 1)
-#define __N               (((PLL0CFG_Val >> 16) & 0x00FF) + 1)
-#define __FCCO(__F_IN)    ((2ULL * __M * __F_IN) / __N)
-#define __CCLK_DIV        (((CCLKCFG_Val      ) & 0x00FF) + 1)
+#define M               (((PLL0CFG_Val      ) & 0x7FFF) + 1)
+#define N               (((PLL0CFG_Val >> 16) & 0x00FF) + 1)
+#define FCCO(__F_IN)    ((2ULL * M * __F_IN) / N)
+#define CCLK_DIV        (((CCLKCFG_Val      ) & 0x00FF) + 1)
```
Comment (authmillenon) [inline boards/mbed_lpc1768/system.c]: False positives?

File: boards/mbed_lpc1768/system.c [hunk 2]
```diff
@@ -382,34 +382,34 @@
...
 #else
 #if   ((CLKSRCSEL_Val & 0x03) == 1)
-#define __CORE_CLK (OSC_CLK         / __CCLK_DIV)
+#define CORE_CLK (OSC_CLK         / __CCLK_DIV)
 #elif ((CLKSRCSEL_Val & 0x03) == 2)
-#define __CORE_CLK (RTC_CLK         / __CCLK_DIV)
+#define CORE_CLK (RTC_CLK         / __CCLK_DIV)
 #else
-#define __CORE_CLK (IRC_OSC         / __CCLK_DIV)
+#define CORE_CLK (IRC_OSC         / __CCLK_DIV)
```
Comment (authmillenon) [inline boards/mbed_lpc1768/system.c]: These ones too?

File: boards/qemu-i386/include/cpu-conf.h [hunk 3]
```diff
@@ -16,8 +16,8 @@
  * @author      René Kijewski <rene.kijewski@fu-berlin.de>
  */
 
-#ifndef __RIOT__BOARDS__QEMU_I386__CPU_CONF__H
-#define __RIOT__BOARDS__QEMU_I386__CPU_CONF__H
+#ifndef RIOT__BOARDS__QEMU_I386__CPU_CONF__H
+#define RIOT__BOARDS__QEMU_I386__CPU_CONF__H
```
Comment (authmillenon) [inline boards/qemu-i386/include/cpu-conf.h]: This guard looks like quite a mouth full.

File: boards/redbee-econotag/tools/ftditools/bbmc.c [hunk 4]
```diff
@@ -353,7 +353,7 @@ int main(int argc, char **argv)
         }
 
         if (ret)
-#endif // __APPLE__
+#endif // APPLE__
```
Comment (authmillenon) [inline boards/redbee-econotag/tools/ftditools/bbmc.c]: False positive

File: boards/z1/include/board.h [hunk 5]
```diff
@@ -36,8 +36,8 @@
 extern "C" {
 #endif
 
-#ifndef __MSP430F2617__
-#define __MSP430F2617__
+#ifndef MSP430F2617__
+#define MSP430F2617__
```
Comment (authmillenon) [inline boards/z1/include/board.h]: This also does not seem to be a guard.

File: boards/mbed_lpc1768/system.c (hunk 1 above)
Comment (OlegHahm) [inline boards/mbed_lpc1768/system.c]: Because it's not a header guard? But we don't want leading underscores in any macro, right? So, I would just update the commit message.

File: boards/qemu-i386/include/cpu-conf.h (hunk 3 above)
Comment (OlegHahm) [inline boards/qemu-i386/include/cpu-conf.h]: Agreed, but changing this is not part of this PR.

File: boards/z1/include/board.h (hunk 5 above)
Comment (OlegHahm) [inline boards/z1/include/board.h]: Overlooked this one.

File: boards/mbed_lpc1768/system.c (hunk 1 above)
Comment (authmillenon) [inline boards/mbed_lpc1768/system.c]: Partly, yes, partly because I have now idea about `mbed_lpc1768`. These look like board-lib vars.

Comment (OlegHahm) [inline boards/mbed_lpc1768/system.c]: They are defined here.

File: boards/chronos/drivers/include/display_putchar.h [hunk 6]
```diff
@@ -6,8 +6,8 @@
  * directory for more details.
  */
 
-#ifndef __DISPLAY_PUTCHAR_H
-#define __DISPLAY_PUTCHAR_H
+#ifndef DISPLAY_PUTCHAR_H
+#define DISPLAY_PUTCHAR_H
```
Comment (haukepetersen) [inline boards/chronos/drivers/include/display_putchar.h]: for all other files until here you appended and underscore to the guard. Wouldn't it make sense to do it one way OR the other -> keep it consistent...

File: boards/mbed_lpc1768/system.c (hunk 1 above)
Comment (haukepetersen) [inline boards/mbed_lpc1768/system.c]: +1 for this change.

File: boards/msb-430/include/board.h [hunk 7]
```diff
@@ -23,8 +23,8 @@
  * @author      Freie Universität Berlin, Computer Systems & Telematics, FeuerWhere project
  */
 
-#ifndef _MSB_BOARD_H
-#define _MSB_BOARD_H
+#ifndef MSB_BOARD_H
+#define MSB_BOARD_H
```
Comment (haukepetersen) [inline boards/msb-430/include/board.h]: trailing underscore

File: boards/msb-430h/include/board.h [hunk 8]
```diff
@@ -18,8 +18,8 @@
  * @author      Freie Universität Berlin, Computer Systems & Telematics, FeuerWhere project
  */
 
-#ifndef _MSB_BOARD_H
-#define _MSB_BOARD_H
+#ifndef MSB_BOARD_H
+#define MSB_BOARD_H
```
Comment (haukepetersen) [inline boards/msb-430h/include/board.h]: trailing underscore and you forgot the comment in the last line of the file.

File: boards/msb-430h/include/periph_conf.h [hunk 9]
```diff
@@ -16,9 +16,10 @@
...
  */
 
-#ifndef PERIPH_CONF_H
-#define PERIPH_CONF_H
+#ifndef PERIPH_CONF_H_
+#define PERIPH_CONF_H_
 
 /* make Travis happy: #ifdef __cplusplus extern "C" { #endif */
 
+#endif /* PERIPH_CONF_H_ */
```
Comment (haukepetersen) [inline boards/msb-430h/include/periph_conf.h]: two `#endif`s?

File: boards/msba2-common/drivers/include/uart0.h [hunk 10]
```diff
@@ -6,8 +6,8 @@
  * directory for more details.
  */
 
-#ifndef __UART0_H
-#define __UART0_H
+#ifndef UART0_H
+#define UART0_H
```
Comment (haukepetersen) [inline boards/msba2-common/drivers/include/uart0.h]: trailing underscore missing

File: boards/msba2-common/include/msba2_common.h [hunk 11]
```diff
@@ -41,4 +41,4 @@ static inline void pllfeed(void)
 #endif
 
 /** @} */
-#endif // __MSBA2_COMMON_H
+#endif // MSBA2_COMMON_H
```
Comment (haukepetersen) [inline boards/msba2-common/include/msba2_common.h]: it's not related to this PR, but you could convert it to c-style comment here.

File: boards/msbiot/include/board.h [hunk 12]
```diff
@@ -105,5 +105,5 @@ void board_init(void);
 }
 #endif
 
-#endif /** __BOARD_H */
+#endif /** BOARD_H_ */
```
Comment (haukepetersen) [inline boards/msbiot/include/board.h]: the comment style should be `/* xxx */` here

File: boards/pca10005/include/board.h [hunk 13]
```diff
@@ -73,5 +73,5 @@ void board_init(void);
 }
 #endif
 
-#endif /** __BOARD_H */
+#endif /** BOARD_H_ */
```
Comment (haukepetersen) [inline boards/pca10005/include/board.h]: care to change to `#endif /* BOARD_H_ */`?

File: boards/pttu/include/board.h [hunk 14]
```diff
@@ -18,8 +18,8 @@
  * @author      Kaspar Schleiser <kaspar@schleiser.de>
  */
 
-#ifndef __BOARD_H
-#define __BOARD_H
+#ifndef BOARD_H
+#define BOARD_H
```
Comment (haukepetersen) [inline boards/pttu/include/board.h]: trailing underscore missing

File: boards/stm32f3discovery/include/board.h [hunk 15]
```diff
@@ -107,5 +107,5 @@ void board_init(void);
 }
 #endif
 
-#endif /** __BOARD_H */
+#endif /** BOARD_H_ */
```
Comment (haukepetersen) [inline boards/stm32f3discovery/include/board.h]: comment style

File: boards/stm32f4discovery/include/board.h [hunk 16]
```diff
@@ -93,5 +93,5 @@ void board_init(void);
 }
 #endif
 
-#endif /** __BOARD_H */
+#endif /** BOARD_H_ */
```
Comment (haukepetersen) [inline boards/stm32f4discovery/include/board.h]: comment style

File: boards/telosb/include/board.h [hunk 17]
```diff
@@ -23,8 +23,8 @@
  * @author      Oliver Hahm <oliver.hahm@inria.fr>
  */
 
-#ifndef _TELOSB_BOARD_H
-#define _TELOSB_BOARD_H
+#ifndef TELOSB_BOARD_H
+#define TELOSB_BOARD_H
```
Comment (haukepetersen) [inline boards/telosb/include/board.h]: trailing underscore missing

File: boards/udoo/include/board.h [hunk 18]
```diff
@@ -82,5 +82,5 @@ void board_init(void);
 }
 #endif
 
-#endif /** __BOARD_H */
+#endif /** BOARD_H_ */
```
Comment (haukepetersen) [inline boards/udoo/include/board.h]: comment style

File: boards/wsn430-v1_3b/include/board.h [hunk 19]
```diff
@@ -23,8 +23,8 @@
  * @author      Milan Babel <babel@inf.fu-berlin.de>
  */
 
-#ifndef _WSN_BOARD_H
-#define _WSN_BOARD_H
+#ifndef WSN_BOARD_H
+#define WSN_BOARD_H
```
Comment (haukepetersen) [inline boards/wsn430-v1_3b/include/board.h]: trailing underscore missing

File: boards/wsn430-v1_4/include/board.h [hunk 20]
```diff
@@ -23,8 +23,8 @@
  * @author      Milan Babel <babel@inf.fu-berlin.de>
  */
 
-#ifndef _WSN_BOARD_H
-#define _WSN_BOARD_H
+#ifndef WSN_BOARD_H
+#define WSN_BOARD_H
```
Comment (haukepetersen) [inline boards/wsn430-v1_4/include/board.h]: trailing underscore missing

File: boards/yunjia-nrf51822/include/board.h [hunk 21]
```diff
@@ -72,5 +72,5 @@ void board_init(void);
 }
 #endif
 
-#endif /** __BOARD_H */
+#endif /** BOARD_H_ */
```
Comment (haukepetersen) [inline boards/yunjia-nrf51822/include/board.h]: comment style

File: boards/z1/include/board.h [hunk 22]
```diff
@@ -7,8 +7,8 @@
  * directory for more details.
  */
 
-#ifndef _Z1_BOARD_H
-#define _Z1_BOARD_H
+#ifndef Z1_BOARD_H
+#define Z1_BOARD_H
```
Comment (haukepetersen) [inline boards/z1/include/board.h]: trailing underscore missing

Comment (haukepetersen) [inline boards/z1/include/board.h]: and you forgot the comment in the end of the file.

**Comment by haukepetersen** [comment]
found some more forgotten stuff and inconsistencies. Also I marked some parts with broken comment styling. I know they are not really part of this PR, but why not fix them while you are at it...

File: boards/msb-430h/include/board.h (hunk 8 above)
Comment (OlegHahm) [inline boards/msb-430h/include/board.h]: All I did was `git grep -l 'BOARD_H' | xargs sed -i 's/BOARD_H/BOARD_H/' and the same for `PERIPH_CONF_H`. But yes, I can change the rest, too.

**Comment by OlegHahm** [comment]
Addressed comments.

**Comment by haukepetersen** [comment]
Looks good to me. ACK when Travis is happy.

**Comment by authmillenon** [comment]
(Needs squashing first)

**Comment by OlegHahm** [comment]
Squashed (no merge conflicts)

**Comment by authmillenon** [comment]
Please squash again, ACK upholds

**Comment by OlegHahm** [comment]
https://www.youtube.com/watch?v=v2H8FtlrzCI

**Comment by authmillenon** [comment]
:D

File: boards/mbed_lpc1768/system.c (hunk 1 above)
Comment (gebart) [inline boards/mbed_lpc1768/system.c]: `#define`-ing one letter names like `N` etc leads to incredibly difficult to understand compile problems when a new user writes their program to use the variable `N` for something.

Comment (gebart) [inline boards/mbed_lpc1768/system.c]: nevermind, this is inside a .c file, not a header.

Comment (OlegHahm) [inline boards/mbed_lpc1768/system.c]: Yes, I took this into account. :)

**Comment by OlegHahm** [comment]
rebased

File: boards/avsextrem/include/smb380-board.h [hunk 23]
```diff
@@ -47,8 +47,8 @@ enum SMB380_MODE {
     SMB380_THRESHOLD,
     SMB380_FALSEALERT
 };
-//volatile enum SMB380_MODE
-volatile enum SMB380_MODE smb380_mode;// = SMB380_POLL;
+/* volatile enum SMB380_MODE */
```
Comment (gebart) [inline boards/avsextrem/include/smb380-board.h]: This line could be deleted instead IMO.

File: boards/avsextrem/include/ssp0-board.h [hunk 24]
```diff
@@ -19,8 +19,8 @@
  *
  * @note        $Id:  avsextrem-ssp0.c  3854 2010-01-18 15:27:01Z zkasmi $
  */
-#ifndef __SSP_H__
-#define __SSP_H__
+#ifndef SSP_H__
```
Comment (gebart) [inline boards/avsextrem/include/ssp0-board.h]: double trailing underscore -> single trailing underscore

File: boards/msb-430h/include/board.h [hunk 25]
```diff
@@ -57,4 +57,4 @@ extern "C" {
 typedef uint8_t radio_packet_length_t;
 
 /** @} */
-#endif // _MSB_BOARD_H
+#endif // MSB_BOARD_H_
```
Comment (gebart) [inline boards/msb-430h/include/board.h]: C++ style comment

File: boards/msba2-common/drivers/include/sht11-board.h [hunk 26]
```diff
@@ -31,13 +31,13 @@
...
-#define SHT11_DATA_HIGH (FIO1SET = BIT26);  // serial I/O line high
-#define SHT11_DATA_IN   (FIO1DIR &= ~BIT26)                     // serial I/O as input
-#define SHT11_DATA_OUT  (FIO1DIR |= BIT26)                      // serial I/O as output
+#define SHT11_SCK_LOW   FIO1CLR = BIT25;    /*  serial clock line low */
+#define SHT11_SCK_HIGH  FIO1SET = BIT25;    /*  serial clock line high */
+#define SHT11_DATA      ((FIO1PIN & BIT26) != 0)                /*  read serial I/O */
+#define SHT11_DATA_LOW  (FIO1CLR = BIT26);  /*  serial I/O line low */
+#define SHT11_DATA_HIGH (FIO1SET = BIT26);  /*  serial I/O line high */
+#define SHT11_DATA_IN   (FIO1DIR &= ~BIT26)                     /*  serial I/O as input */
+#define SHT11_DATA_OUT  (FIO1DIR |= BIT26)                      /*  serial I/O as output */
```
Comment (gebart) [inline boards/msba2-common/drivers/include/sht11-board.h]: align comment indentations above

File: boards/msba2-common/tools/src/boot.h [hunk 27]
```diff
@@ -1,5 +1,5 @@
 /*
- * LPC 2000 Loader, http://www.pjrc.com/arm/lpc2k_pgm
+ * LPC 2000 Loader, http:/* www.pjrc.com/arm/lpc2k_pgm */
```
Comment (gebart) [inline boards/msba2-common/tools/src/boot.h]: Too greedy C++ comment replacement

File: boards/msba2-common/tools/src/boot_23xx.h [hunk 28]
```diff
@@ -1,5 +1,5 @@
 /*
- * LPC 2000 Loader, http://www.pjrc.com/arm/lpc2k_pgm
+ * LPC 2000 Loader, http:/* www.pjrc.com/arm/lpc2k_pgm */
```
Comment (gebart) [inline boards/msba2-common/tools/src/boot_23xx.h]: revert

File: boards/msba2-common/tools/src/boot_2xxx.h [hunk 29]
```diff
@@ -1,5 +1,5 @@
 /*
- * LPC 2000 Loader, http://www.pjrc.com/arm/lpc2k_pgm
+ * LPC 2000 Loader, http:/* www.pjrc.com/arm/lpc2k_pgm */
```
Comment (gebart) [inline boards/msba2-common/tools/src/boot_2xxx.h]: revert

File: boards/msba2-common/tools/src/chipinfo.h [hunk 30]
```diff
@@ -1,5 +1,5 @@
 /*
- * LPC 2000 Loader, http://www.pjrc.com/arm/lpc2k_pgm
+ * LPC 2000 Loader, http:/* www.pjrc.com/arm/lpc2k_pgm */
```
Comment (gebart) [inline boards/msba2-common/tools/src/chipinfo.h]: revert

File: boards/msba2-common/tools/src/chipinfo.h [hunk 31]
```diff
@@ -18,19 +18,19 @@
...
-    int num_sector;     // number of flash sectors
-    struct sector_info_struct *layout;  // layout of sectors
-    const unsigned int *bootprog;   // code that boots into user program (NULL = DTR/RTS only)
+    char *part_number;  /*  human readable part number */
+    char *id_string;    /*  id string sent by "J" command */
+    unsigned int ram_addr;  /*  where to download into RAM */
+    int chunk_size;     /*  download to ram chunk size */
+    int num_sector;     /*  number of flash sectors */
+    struct sector_info_struct *layout;  /*  layout of sectors */
+    const unsigned int *bootprog;   /*  code that boots into user program (NULL = DTR/RTS only) */
```
Comment (gebart) [inline boards/msba2-common/tools/src/chipinfo.h]: align comment indentation in block above

File: boards/msba2-common/tools/src/chipinfo.h [hunk 32]
```diff
@@ -18,19 +18,19 @@
 
 extern char *lpc_return_strings[];
 
-struct sector_info_struct { // an array of
-    int address;        // where each sector is located
-    int size;       // and how big it is
+struct sector_info_struct { /*  an array of */
+    int address;        /*  where each sector is located */
+    int size;       /*  and how big it is */
```
Comment (gebart) [inline boards/msba2-common/tools/src/chipinfo.h]: align comment indentation above

File: boards/msba2-common/tools/src/control_2xxx.h [hunk 33]
```diff
@@ -1,5 +1,5 @@
 /*
- * LPC 2000 Loader, http://www.pjrc.com/arm/lpc2k_pgm
+ * LPC 2000 Loader, http:/* www.pjrc.com/arm/lpc2k_pgm */
```
Comment (gebart) [inline boards/msba2-common/tools/src/control_2xxx.h]: revert

File: boards/msba2-common/tools/src/control_2xxx.h [hunk 34]
```diff
@@ -16,10 +16,10 @@
  * 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
  */
 
-#ifndef CONTROL_2XXXX_H
-#define CONTROL_2XXXX_H
+#ifndef CONTROL_2XXXX_H_
+#define CONTROL_2XXXX_H_
```
Comment (gebart) [inline boards/msba2-common/tools/src/control_2xxx.h]: `s/2XXXX/2XXX/`

File: boards/msba2-common/tools/src/control_2xxx.h [hunk 35]
```diff
@@ -16,10 +16,10 @@
...
 
 void hard_reset_to_bootloader(void);
 void hard_reset_to_user_code(void);
 
-#endif // ..._H
+#endif /*  ..._H_ */
```
Comment (gebart) [inline boards/msba2-common/tools/src/control_2xxx.h]: `/* CONTROL_2XXX_H_ */`

File: boards/msba2-common/tools/src/download.h [hunk 36]
```diff
@@ -1,5 +1,5 @@
 /*
- * LPC 2000 Loader, http://www.pjrc.com/arm/lpc2k_pgm
+ * LPC 2000 Loader, http:/* www.pjrc.com/arm/lpc2k_pgm */
```
Comment (gebart) [inline boards/msba2-common/tools/src/download.h]: revert

File: boards/msba2-common/tools/src/gui.h [hunk 37]
```diff
@@ -1,5 +1,5 @@
 /*
- * LPC 2000 Loader, http://www.pjrc.com/arm/lpc2k_pgm
+ * LPC 2000 Loader, http:/* www.pjrc.com/arm/lpc2k_pgm */
```
Comment (gebart) [inline boards/msba2-common/tools/src/gui.h]: revert

File: boards/msba2-common/tools/src/ihex.h [hunk 38]
```diff
@@ -1,5 +1,5 @@
 /*
- * LPC 2000 Loader, http://www.pjrc.com/arm/lpc2k_pgm
+ * LPC 2000 Loader, http:/* www.pjrc.com/arm/lpc2k_pgm */
```
Comment (gebart) [inline boards/msba2-common/tools/src/ihex.h]: revert

File: boards/msba2-common/tools/src/lpc2k_pgm.h [hunk 39]
```diff
@@ -1,5 +1,5 @@
 /*
- * LPC 2000 Loader, http://www.pjrc.com/arm/lpc2k_pgm
+ * LPC 2000 Loader, http:/* www.pjrc.com/arm/lpc2k_pgm */
```
Comment (gebart) [inline boards/msba2-common/tools/src/lpc2k_pgm.h]: revert

File: boards/msba2-common/tools/src/lpc2k_pgm.h [hunk 40]
```diff
@@ -27,4 +27,4 @@ void change_baud(const char *baud_name);
 */
 void signal_terminal(void);
 
-#endif // LPC2K_PGM
+#endif /*  LPC2K_PGM */
```
Comment (gebart) [inline boards/msba2-common/tools/src/lpc2k_pgm.h]: Might as well change the include guard name in this file

File: boards/msba2-common/tools/src/serial.h [hunk 41]
```diff
@@ -1,5 +1,5 @@
 /*
- * LPC 2000 Loader, http://www.pjrc.com/arm/lpc2k_pgm
+ * LPC 2000 Loader, http:/* www.pjrc.com/arm/lpc2k_pgm */
```
Comment (gebart) [inline boards/msba2-common/tools/src/serial.h]: revert

File: boards/msba2-common/tools/src/settings.h [hunk 42]
```diff
@@ -1,5 +1,5 @@
 /*
- * LPC 2000 Loader, http://www.pjrc.com/arm/lpc2k_pgm
+ * LPC 2000 Loader, http:/* www.pjrc.com/arm/lpc2k_pgm */
```
Comment (gebart) [inline boards/msba2-common/tools/src/settings.h]: revert

File: boards/msba2-common/tools/src/uuencode.h [hunk 43]
```diff
@@ -1,5 +1,5 @@
 /*
- * LPC 2000 Loader, http://www.pjrc.com/arm/lpc2k_pgm
+ * LPC 2000 Loader, http:/* www.pjrc.com/arm/lpc2k_pgm */
```
Comment (gebart) [inline boards/msba2-common/tools/src/uuencode.h]: revert

File: boards/redbee-econotag/include/board.h [hunk 44]
```diff
@@ -27,7 +27,7 @@
 extern "C" {
 #endif
 
-#define F_CPU   (24000000)              ///< CPU target speed in Hz
+#define F_CPU   (24000000)              /* /< CPU target speed in Hz */
```
Comment (gebart) [inline boards/redbee-econotag/include/board.h]: Doxygen opening tag messed up

File: boards/qemu-i386/include/board.h [hunk 45]
```diff
@@ -18,8 +18,8 @@
  * @author      René Kijewski <rene.kijewski@fu-berlin.de>
  */
 
-#ifndef __RIOT__BOARDS__QEMU_I386__BOARD__H
-#define __RIOT__BOARDS__QEMU_I386__BOARD__H
+#ifndef RIOT__BOARDS__QEMU_I386__BOARD__H_
+#define RIOT__BOARDS__QEMU_I386__BOARD__H_
```
Comment (gebart) [inline boards/qemu-i386/include/board.h]: rename this, double underscores are not allowed (also it does not follow the pattern of the rest of the headers)

File: boards/qemu-i386/include/cpu_conf.h [hunk 46]
```diff
@@ -16,8 +16,8 @@
  * @author      René Kijewski <rene.kijewski@fu-berlin.de>
  */
 
-#ifndef __RIOT__BOARDS__QEMU_I386__CPU_CONF__H
-#define __RIOT__BOARDS__QEMU_I386__CPU_CONF__H
+#ifndef RIOT__BOARDS__QEMU_I386__CPU_CONF__H_
+#define RIOT__BOARDS__QEMU_I386__CPU_CONF__H_
```
Comment (gebart) [inline boards/qemu-i386/include/cpu_conf.h]: rename

File: boards/wsn430-v1_3b/include/board.h [hunk 47]
```diff
@@ -72,4 +72,4 @@ extern "C" {
 typedef uint8_t radio_packet_length_t;
 
 /** @} */
-#endif // _WSN_BOARD_H
+#endif // WSN_BOARD_H_
```
Comment (gebart) [inline boards/wsn430-v1_3b/include/board.h]: C++ style comment

File: boards/x86-multiboot-common/include/multiboot.h [hunk 48]
```diff
@@ -25,8 +25,8 @@
  * @brief       Architecture specific definitions for multiboot enabled kernels
  */
 
-#ifndef MULTIBOOT_H
-#define MULTIBOOT_H
+#ifndef MULTIBOOT_H_
+#define MULTIBOOT_H_
```
Comment (gebart) [inline boards/x86-multiboot-common/include/multiboot.h]: The corresponding `#endif` is not modified properly.

**Comment by gebart** [comment]
I went through all changes and commented where I could spot any mistakes. This should be ready after my comments are addressed.

File: boards/qemu-i386/include/board.h (hunk 45 above)
Comment (OlegHahm) [inline boards/qemu-i386/include/board.h]: I can remove the double underscores, but I won't rename completely because I fear a potential clash with qemu headers.

**Comment by OlegHahm** [comment]
addressed comments

File: boards/qemu-i386/include/board.h (hunk 45 above)
Comment (gebart) [inline boards/qemu-i386/include/board.h]: I am fine with that solution.

Comment (Kijewski) [inline boards/qemu-i386/include/board.h]: No QEMU headers are used.

Comment (OlegHahm) [inline boards/qemu-i386/include/board.h]: ok, updated then

**Comment by authmillenon** [comment]
Please squash.

File: boards/chronos/drivers/include/display_putchar.h [hunk 49]
```diff
@@ -19,4 +19,4 @@ void init_display_putchar(void);
 }
 #endif
 
-#endif /* __DISPLAY_PUTCHAR_H */
+#endif /* DISPLAY_PUTCHAR_H */
```
Comment (gebart) [inline boards/chronos/drivers/include/display_putchar.h]: wrong macro name in comment

File: boards/mbed_lpc1768/system.c [hunk 50]
```diff
@@ -382,34 +382,34 @@
 
 
 /* F_cco0 = (2 * M * F_in) / N  */
-#define __M               (((PLL0CFG_Val      ) & 0x7FFF) + 1)
-#define __N               (((PLL0CFG_Val >> 16) & 0x00FF) + 1)
-#define __FCCO(__F_IN)    ((2ULL * __M * __F_IN) / __N)
-#define __CCLK_DIV        (((CCLKCFG_Val      ) & 0x00FF) + 1)
+#define M               (((PLL0CFG_Val      ) & 0x7FFF) + 1)
+#define N               (((PLL0CFG_Val >> 16) & 0x00FF) + 1)
+#define FCCO(__F_IN)    ((2ULL * M * __F_IN) / N)
```
Comment (gebart) [inline boards/mbed_lpc1768/system.c]: Nitpick: Where is `__F_IN` defined? Can it be changed too?

**Comment by gebart** [comment]
I can only find small nitpicks now. I think this is ready for merging. ACK, please squash. Merge when Travis is happy.

**Comment by OlegHahm** [comment]
Addressed @gebart's nitpicking. :wink:

**Comment by gebart** [comment]
ACK, squash and go when Travis is fine.

**Comment by OlegHahm** [comment]
Squashed.

**Comment by gebart** [comment]
kicked Travis

**Comment by Kijewski** [comment]
ACK
